sql/cagg_migrate.sql

```diff
@@ -592,3 +592,9 @@ BEGIN
     WHERE mat_hypertable_id OPERATOR(pg_catalog.=) _cagg_data.mat_hypertable_id;
 END;
 $BODY$;
+
+
+CREATE OR REPLACE FUNCTION _timescaledb_functions.cagg_recovery_get_bucket_function(
+    mat_hypertable_id INTEGER
+) RETURNS regprocedure AS '@MODULE_PATHNAME@', 'ts_continuous_agg_recovery_get_bucket_function' LANGUAGE C STRICT VOLATILE;
```

Comment:
Why adding it to the extension?? I mean we'll need it just for the update, right?? If yes then we can create and drop it in latest-dev.sql.

And if the intention is make it permanent I guess this is not the best place since here we have all the cagg migration related code. Maybe the best place for it should be `sql/cagg_utils.sql`.

What about a simpler name like `cagg_get_bucket_function` ???

Reply:
Good point. My idea was to re-use it in a later migration. But I could re-register the function if needed. 

So, the function is removed from the extension and renamed.